### ISSUE TITLE
Errors closing doc

### ISSUE THREAD

**Issue by chadrolfs** [adobe-photoshop/spaces-design#1684]
new default doc, close it.  Result: 
`Action documents.close failed: TypeError: Cannot read property 'delete' of undefined`
`Uncaught TypeError: Cannot read property 'update' of undefined`
`Uncaught TypeError: Cannot read property 'delete' of undefined`
`Action application.afterStartup failed: TypeError: Cannot read property '_currentElement' of null`

**Comment by chadrolfs** [comment]
a76ba6bdd251a11e1bb6c153bc5232262c317775 is the first bad commit
commit a76ba6bdd251a11e1bb6c153bc5232262c317775
Author: Ian Wehrman <wehrman@adobe.com>

**Comment by chadrolfs** [comment]
```Action documents.close failed: TypeError: Cannot read property 'delete' of undefined
    at Fluxxor.createStore._clearDroppables (file:///Users/crolfs/playground-repos/spaces-design/src/js/stores/draganddrop.js:162:37)
    at Fluxxor.createStore.batchDeregisterDroppables (file:///Users/crolfs/playground-repos/spaces-design/src/js/stores/draganddrop.js:207:22)
    at React.createClass.componentWillUnmount (file://./js/jsx/sections/layers/LayersPanel.jsx:167:39)
    at chainedFunction [as componentWillUnmount] (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:6044:9)
    at ReactCompositeComponentMixin.unmountComponent (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:6929:14)
    at Object.ReactReconciler.unmountComponent (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:14799:22)
    at Object.ReactChildReconciler.unmountChildren (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:5246:23)
    at ReactDOMComponent.ReactMultiChild.Mixin.unmountChildren (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:13551:28)
    at ReactDOMComponent.Mixin.unmountComponent (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:8399:10)
    at Object.ReactReconciler.unmountComponent (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:14799:22)
    at ReactCompositeComponentMixin.unmountComponent (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:6935:21)
    at Object.ReactReconciler.unmountComponent (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:14799:22)
    at ReactCompositeComponentMixin.unmountComponent (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:6935:21)
    at Object.ReactReconciler.unmountComponent (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:14799:22)
    at Object.ReactChildReconciler.unmountChildren (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:5246:23)
    at ReactDOMComponent.ReactMultiChild.Mixin.unmountChildren (file:///Users/crolfs/playground-repos/spaces-design/bower_components/react/react-with-addons.js:13551:28)
From previous event:
    at Object.updateRecentFiles (file:///Users/crolfs/playground-repos/spaces-design/src/js/actions/application.js:59:14)
    at FluxController._applyAction (file:///Users/crolfs/playground-repos/spaces-design/src/js/fluxcontroller.js:290:36)
    at Object.create.transfer.value (file:///Users/crolfs/playground-repos/spaces-design/src/js/fluxcontroller.js:227:33)
    at Object.<anonymous> (file:///Users/crolfs/playground-repos/spaces-design/src/js/actions/documents.js:439:47)
From previous event:
    at Object.disposeDocument (file:///Users/crolfs/playground-repos/spaces-design/src/js/actions/documents.js:429:14)
    at FluxController._applyAction (file:///Users/crolfs/playground-repos/spaces-design/src/js/fluxcontroller.js:290:36)
    at Object.create.transfer.value (file:///Users/crolfs/playground-repos/spaces-design/src/js/fluxcontroller.js:227:33)
    at Object.<anonymous> (file:///Users/crolfs/playground-repos/spaces-design/src/js/actions/documents.js:321:29)
From previous event:
    at Object.close (file:///Users/crolfs/playground-repos/spaces-design/src/js/actions/documents.js:320:14)
    at FluxController._applyAction (file:///Users/crolfs/playground-repos/spaces-design/src/js/fluxcontroller.js:290:36)
    at null.<anonymous> (file:///Users/crolfs/playground-repos/spaces-design/src/js/fluxcontroller.js:382:37)
From previous event:
    at null.<anonymous> (file:///Users/crolfs/playground-repos/spaces-design/src/js/fluxcontroller.js:378:22)
    at AsyncDependencyQueue._processNext (file:///Users/crolfs/playground-repos/spaces-design/src/js/util/async-dependency-queue.js:292:27)
    at AsyncDependencyQueue.push (file:///Users/crolfs/playground-repos/spaces-design/src/js/util/async-dependency-queue.js:144:14)
    at Object.<anonymous> (file:///Users/crolfs/playground-repos/spaces-design/src/js/fluxcontroller.js:364:42)
    at Object.throttledFn (file:///Users/crolfs/playground-repos/spaces-design/src/js/util/synchronization.js:66:31)
    at Object.<anonymous> (file:///Users/crolfs/playground-repos/spaces-design/src/js/actions/menu.js:270:13)
    at emitEvent (file:///Users/crolfs/playground-repos/spaces-design/bower_components/eventEmitter/EventEmitter.js:380:50)
    at emit (file:///Users/crolfs/playground-repos/spaces-design/bower_components/eventEmitter/EventEmitter.js:407:21)
    at UI._menuEventHandler (file:///Users/crolfs/playground-repos/spaces-design/bower_components/spaces-adapter/src/ps/ui.js:90:14)```

**Comment by chadrolfs** [comment]
or if this is easier to read:
![screen shot 2015-07-07 at 2 39 52 pm](https://cloud.githubusercontent.com/assets/8442631/8557956/c6b9da7a-24b5-11e5-9c9c-4f9e808a36a1.png)

**Comment by iwehrman** [comment]
I pushed a tiny fix directly to master with 65235e273d3ced4711ec377f20bfe8d13659b61a. FBNC to @chadrolfs.
